changelogs/unreleased/gh-4693-disable-argument-list-syntax-of-box-tuple-new.md

```diff
@@ -0,0 +1,4 @@
+## feature/box
+
+* **[Breaking change]** Disabled argument list syntax of `box.tuple.new` under
+  compatibility (this was needed for gh-4693).
```

Comment:
Nit: "under compatibility" sounds confusing. Better say something like "Diabled argument list syntax of `box.tuple.new`. It is possible to switch to the old behavior using the compatibility option `option-name`".

Reply:
Applied your suggestion.